gov2/lambda/CreateFunction/CreateFunction.go

```diff
@@ -0,0 +1,117 @@
+//Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\
+//SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/lambda"
+	"github.com/aws/aws-sdk-go-v2/service/lambda/types"
+)
+
+//This example creates a Lambda Function in your AWS Account using the Go V2 SDK. Note that it requies you
+// to have a zip file of the actual code already created and uploaded to an S3 Bucket.
+func main() {
+
+	//Your unique AWS Account Id
+	awsAccountId := os.Getenv("AWS_ACCOUNT_ID")
```

Comment:
Use input flags instead of env variables. Or you can get the account ID from the STS service.